src/main/java/frc/robot/commands/IntakeDeployCommand.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.IntakeDeploySubsystem;
+
+public class IntakeDeployCommand extends CommandBase {
+    IntakeDeploySubsystem intakeDeploySubsystem;
+    BooleanSupplier position;
```

Comment:
```suggestion
    private final IntakeDeploySubsystem intakeDeploySubsystem;
    private final BooleanSupplier position;
```

Reply:
Is there a reason we need a dedicated command for this anyway? We could probably do something like this instead:
```java
Command deployIntakeCommand = new InstantCommand(() -> intakeSubsystem.setSolenoid(true), intakeSubsystem);
Command retractIntakeCommand = new InstantCommand(() -> intakeSubsystem.setSolenoid(false), intakeSubsystem);
```
And then we bind pushing the button to the first command and releasing it to the second command. This also gives us flexibility in autonomous to keep the intake deployed while running other commands, then retracting it later.

Reply:
With the way this command is currently written, I'm not totally sure it will fit well with autonomous (`isFinished()` returns `false`, so the next command will never run).